package.json

```diff
@@ -26,7 +26,7 @@
     "clean": "make clean",
     "help": "make help",
     "refresh": "make refresh",
-    "test": "make test"
+    "test": "mocha -R list"
```

Comment:
I usually use the simplest form, which is `"test": "mocha"`. Mocha picks `test/*` by default, which is good enough in most cases.

Note that some of our repositories (most notably juggler and loopback) abuse the fact that mocha picks non-test files too, put helpers to `test/support.js` and expose them on global scope.  In that case, `mocha test/*.test.js*` would cause test failures.